pinecone/control/pinecone.py

```diff
@@ -561,15 +575,28 @@ def configure_index(
 
         """
         api_instance = self.index_api
-        config_args: Dict[str, Any] = {}
+
+        if deletion_protection is None:
+            description = self.describe_index(name=name)
+            dp = DeletionProtection(description.deletion_protection)
```

Comment:
If the user does not specify a value for deletion_protection, we call describe_index to look up the current value. This is needed to avoid disabling deletion protection when somebody is trying to call this with `replicas` or `pod_type` only.